src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -142,6 +144,14 @@ public WebElement getHorizontalItemValue() {
     return horizontalItemValue;
   }
 
+  public WebElement getNavigationElements(int index) { return navigationElements.get(index); }
+
+  public WebElement getGroupHeader(int index) { return groupHeadersWrappers.get(index); }
+
+  public WebElement getImageWrapper() { return imageWrapper; }
+
+  public WebElement getTitleWrapper() { return titleWrapper; }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)